src/Database/Migration/MigrationRepository.php

```diff
@@ -49,4 +49,15 @@ public function setMigrator(Migrator $migrator)
 
         return $this;
     }
+
+    /**
+     * Remove a migration from the log.
+     *
+     * @param  object  $migration
+     * @return void
+     */
+    public function delete($migration)
+    {
+        $this->table()->where('migration','LIKE', '%'.$migration->getAddon()->getNamespace().'%')->delete();
```

Comment:
Like is case sensitive, Do you always get the namespace exactly right?

Reply:
Yes. The format is strict.